Sources/AppKit/NSTableView.swift

```diff
@@ -18,6 +18,22 @@ public extension ReactiveExtensions where Base: NSTableView {
   public var dataSource: ProtocolProxy {
     return base.protocolProxy(for: NSTableViewDataSource.self, setter: NSSelectorFromString("setDataSource:"))
   }
+
+  public var selectionWillChange: SafeSignal<Void> {
+    return NotificationCenter.default.reactive.notification(name: .NSTableViewSelectionWillChange).eraseType()
+  }
+
+  public var selectionDidChange: SafeSignal<Void> {
+    return NotificationCenter.default.reactive.notification(name: .NSTableViewSelectionDidChange).eraseType()
```

Comment:
Oh, god 🤦‍♂️ I've been using this API (NSNotificationCenter) for over a decade, and I still get it wrong…